package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "wormhole-solidity-sdk",
```

Comment:
This package should be published under the @wormhole-foundation scope.

Reply:
Of course, but since I contacted someone there for a while and nobody cared about it, months ago I published an Hardhat version of it by myself, in order to use it in my projects and to allow a couple of friends who had the same issue to work easily inside hardhat. The beauty of open source is that it allows work-around, even when the official published doesn't give a s**t about something :-)

Reply:
Yeah, I agree. I apologize about the neglect here, the ecosystem is still in the process of slowly getting its shit together. But I'm now the new Sheriff in town and this sort of neglect will not happen under my watch.